File: projects/tgs_fanj/tgs_反击/Script/lua/Dynamic/室5盒子.lua
--打开触发
function OnTurnOn(uSource, uDest)
  M_MapIceMonster(70, '室5四臂金刚', false);
  M_MapIceMonster(70, '室5护卫武士', false);
  M_MapboNotHItMonster(70, '室5四臂金刚', false);
  M_MapboNotHItMonster(70, '室5护卫武士', false);
end;

--关闭触发
function OnTurnOff(uSource)
  M_MapIceMonster(70, '室5四臂金刚', true);
  M_MapIceMonster(70, '室5护卫武士', true);
  M_MapboNotHItMonster(70, '室5四臂金刚', true);
  M_MapboNotHItMonster(70, '室5护卫武士', true);
end;

File: projects/1000y/debug/tgs1000/Script/Dynamic/室5盒子.pas
  //打开

procedure OnTurnOn(uSource, uDest: integer);
begin
  MapIceMonster('王陵3层', '室5四臂金刚', false);
  MapIceMonster('王陵3层', '室5护卫武士', false);
end;
 //复活

procedure OnTurnOff(uSource: integer);
begin
  MapIceMonster('王陵3层', '室5四臂金刚', true);
  MapIceMonster('王陵3层', '室5护卫武士', true);
end;

